packages/core/content-type-builder/admin/src/components/SelectComponents/index.js

```diff
@@ -54,7 +54,7 @@ const SelectComponents = ({ dynamicZoneTarget, intlLabel, name, onChange, value
       defaultMessage:
         '{number, plural, =0 {# components} one {# component} other {# components}} selected',
     },
-    { number: value.length }
+    { number: value?.length }
```

Comment:
I don't know the context outside of this, but here 'number' will be undefined, would it be better if it was 0?

Reply:
Yes that's true